src/utils/compoundFunctions/index.js

```diff
@@ -0,0 +1,43 @@
+const { flow } = require("lodash/fp");
+const { parseContent, getInnerHTMLWithPiping } = require("../HTMLUtils");
+const convertPipes = require("../convertPipes");
+
+const processPipe = ctx => flow(convertPipes(ctx), getInnerHTMLWithPiping);
+
+const wrapContents = propName => content => {
+  if (!propName || propName === "" || !content) {
+    return;
+  }
+
+  const result = parseContent(content);
+
+  if (!result) {
+    return;
+  }
+  return { [propName]: result };
+};
+
+const reversePipeContent = ctx => data => {
+  if (!data) {
+    return "";
+  }
```

Comment:
Could we throw an error rather than return an empty string? If not, could we return undefined so that the error states throughout the feature are consistent?